launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/ForkScaffolder.kt

```diff
@@ -24,6 +24,7 @@ class ForkScaffolder(private val launcher: Launcher) {
             instance.default.type={{instanceType}}
             instance.default.runModes={{ localInstanceRunModes }}
             instance.default.password={{instancePassword}}
+            instance.cloud.keyPath={{keyPath}}
```

Comment:
hmmm `instance.cloud.` ?

usually, I am scoping the names of properties to some GAP services like packageManager etc
the instance.[name].[property] is an exceptional convention for defining instances so I am confused here about why it is not an instance.[name].keyPath

Reply:
ok will change cloud to "local-author", I didn't know those names serve, I thought it's only for naming, and the keyPath is only used for the could so made it this way
and btw, why do we call it local-author while this is used for remote?

Reply:
https://github.com/wttech/gradle-aem-plugin/blob/master/docs/common-plugin.md#instance-conventions

Reply:
See instance conventions

Reply:
ok got it, I have it now as local-author but will change it to remote-author which is more indicative

Reply:
You didn't get it. Use 'default' as for password which will provide a value for any instance type

Reply:
oh! ok, on it